check.js

```diff
@@ -11,63 +15,162 @@ var cv1 = pjson.version;
 var cv = 'v' + cv1;
 
 try {
-    config = require("config");
+    config = require("config/parser");
+    config.mergeDeep(config, DEFAULT)
+
 } catch (err) {
     if (String(err).includes("SyntaxError: ")) {
         console.error("The syntax in your config file is not correct. Make sure you replaced all values as the README says under 'How to Install' step 5. If it still does not work, check that all quotes are closed. You can look up the json syntax online. Please note that the comments are no problem although comments are normally not allowed in json. " + err)
         process.exit(1);
     }
 }
-check();
 
-function check() {
+const configPath = path.join(process.cwd(), './config/default.json');
 
-    var updatemessage = config.updatemessage;
-    (async () => {
-        let feed = await parser.parseURL('https://github.com/themoonisacheese/2bored2wait/releases.atom');
-        feed.items.every(item => {
-            var lv = (item.title);
-            if (!cv.includes(lv) && updatemessage != "n") {
-                console.log(boxen('New Update Available! → ' + lv, {
-                    padding: 1,
-                    margin: 1,
-                    align: 'center',
-                    borderColor: 'red',
-                    float: 'center',
-                    borderStyle: 'round'
-                }));
-                rl.question("To continue type 1. To edit settings type 2. ", function(choice) {
-                    if (choice == 1) {
-                        start();
-                    } else if (choice == 2) {
-                        settings();
-                    } else {
-                        console.log("Invalid response.");
-                        check();
-                    };
-                });
-            } else {
-                start();
-            };
-        });
-    })();
+// start the checks
+getconf();
 
-    function start() {
-        console.log("Please wait...");
-        rl.close();
-        require('./main.js');
-    }
+function getconf() {
```

Comment:
It would be better if it pulled from default.json.

Reply:
?

Reply:
I commented in the wrong place. It's the lines below this one.

Reply:
It is pulling from default.conf. Check.js’s new job is to make the config file that way the rest of the binaries to work. Without it we wouldn’t be able to start main.js. If default.json alread exist then it will skip over making the file and continue

Reply:
Alright sounds good.

Reply:
👍